tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -11475,7 +11475,7 @@ process_auto_sptr(SPTR sptr)
   /* Now create the alloca for this variable. Since the alloca produces the
    * address of the local, name it "%foo.addr". */
   local = ll_create_local_object(llvm_info.curr_func, type, align_of_var(sptr),
-                                 "%s.addr", SYMNAME(sptr));
+    "%s.addr", SYMNAME(sptr));
```

Comment:
Should it be 4 spaces instead of 2 spaces before `"%s.addr"`? And why not keep the original code style?

Reply:
Thanks for the comment, my editor screw up flang existing indentation/style, every-time I make change.
For now, I've done 4 spaces change. Is it Okay ?

Reply:
Sorry for the unclear comment. I mean 4 spaces starting from the `l` of `local`. That is, should it be 6 spaces in total?
```
   local = ll_create_local_object(llvm_info.curr_func, type, align_of_var(sptr),
-                                 "%s.addr", SYMNAME(sptr));
+      "%s.addr", SYMNAME(sptr));
```

Reply:
Taken care Thanks @PeixinQiao !

Reply:
Looks good for me now.

Reply:
This looks like an unnecessary diff to me, actually.

Reply:
Thanks for catching @bryanpkc :), like I said my editor is having hard-time understanding flang's indentation :(